README.markdown

```diff
@@ -14,12 +14,14 @@ When developing and deploying your applications you are interacting with two dif
 
 _[Using [Composer](http://getcomposer.org/)]_
 
-Add the plugin to your project's `composer.json` - something like this:
+Run `composer require josegonzalez/dotenv:~1.0`
+
+Or add the plugin to your project's `composer.json` - something like this:
 
 ```composer
   {
     "require": {
-      "josegonzalez/dotenv": "dev-master"
+      "josegonzalez/dotenv": "~1.0"
```

Comment:
Can you revert this, or at least change the docs to state that you can require a specific version if desired. I don't want to have to change this readme file on every release.